WebViewControl/WebView.cs

```diff
@@ -420,7 +422,7 @@ public partial class WebView : IDisposable {
         private JavascriptExecutor GetJavascriptExecutor(string frameName) {
             lock (JsExecutors) {
                 var frameNameForIndex = frameName ?? "";
-                if (!JsExecutors.TryGetValue(frameNameForIndex, out var jsExecutor)) {
+                if (!JsExecutors.TryGetValue(frameNameForIndex, out var jsExecutor) && !isDisposing) {
```

Comment:
As now we are removing jsExecutors in the dispose, we need to avoid creating them again if called after dispose. Not sure if we should lock SyncRoot here before accessing isDisposing as is the locker used to change the isDisposing flag. Asking before adding it because not sure if could affect performance or any operation

Reply:
I see but in that case you should instead do the following on dispose:
```
chromium.JavascriptContextCreated -= OnJavascriptContextCreated;
chromium.JavascriptContextReleased -= OnJavascriptContextReleased;

Reply:
I can do that as well but It's not enough because EvaluateScriptFunction can still be called after dispose which will trigger the creation of a new executor. At least we have tests validating the return value in this scenario.

Reply:
Consider putting the bail out of the !isDisposing either in the beginning of the method or as first condition in this if